### PR TITLE
Ssd 3561 retain filters, clear name, organization menu target

### PR DESCRIPTION
**Pull request by fabian202**

### CODE REVIEW

File: src/components/molecules/DateSelector/index.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+              selected={dateTo}
+              onChange={(date) => handleDateChange(date, 'to')}
+              minDate={dateFrom}
+              showMonthDropdown
+              showYearDropdown
+              onChangeRaw={(e) => e.preventDefault()}
+            />
+          </Col>
+        </Row> */
+}
```
Comment (normandanielmq) [inline src/components/molecules/DateSelector/index.js:126]: Can be removed?

File: src/components/molecules/ProgramsTable/SearchHeader/index.js [hunk 2]
```diff
@@ -1,12 +1,16 @@
 import React, { useState, useCallback, useEffect } from 'react'
-import { Input, OrganizationSelector, UserSelector } from 'components'
+import { OrganizationSelector, DateSelector, UserSelector } from 'components'
 import { debounce } from 'lodash'
-import { StyledSearchHeader } from './style'
+import { StyledSearchHeader, Input } from './style'
+import { SearchContainer, Close } from '../../SearchPanel/style'
```
Comment (normandanielmq) [inline src/components/molecules/ProgramsTable/SearchHeader/index.js:5]: Not sure is a good idea to share style between independent components, can we move them to atoms?

File: src/components/molecules/ProgramsTable/SearchHeader/index.js [hunk 3]
```diff
@@ -72,7 +82,6 @@ const SearchHeader = (
           />
         </th>
       }
-      <th/>
```
Comment (normandanielmq) [inline src/components/molecules/ProgramsTable/SearchHeader/index.js:75]: Are you sure this not needed, it looks like there is a incomplete header section at the very right

Comment (fabian202) [reply src/components/molecules/ProgramsTable/SearchHeader/index.js:75]: Yes, tested on both assignments screens and works good!

**Pull request merged**